spec

```diff
@@ -599,8 +599,12 @@ beginseg
     include "$(BUILD_DIR)/src/code/fmodf.o"
     include "$(BUILD_DIR)/src/code/__osMemset.o"
     include "$(BUILD_DIR)/src/code/__osMemmove.o"
-    include_data_with_rodata "$(BUILD_DIR)/src/code/z_message_PAL.o"
-    include "$(BUILD_DIR)/src/code/z_game_over.o"
+    // For some reason, the data sections of these files are placed here near the
+    // rodata sections of the other files
+    include_data_only_with_rodata "$(BUILD_DIR)/src/code/z_message_PAL.o"
+    include_data_only_with_rodata "$(BUILD_DIR)/src/code/z_game_over.o"
```

Comment:
The directive's name is a bit confusing. Maybe `within_rodata` instead of `with_rodata` would be a bit better? idk